articles/logs/streams/health.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Log Streams Health
+description: All Log Streams types let you view your stream health in the health tab.
+toc: false
+topics:
+ - logs
+ - streams
+ - event-streams
+contentType: how-to
+---
+
+# Viewing an Existing Event Stream's Health
+
+Log Stream Health is the functionality that allows customers to troubleshoot issues with Log Streams. It allows customers to get the status of the latest delivery attempts of each stream to make it easy to identify problems in the delivery.
+
+## Viewing an Existing Event Stream's Health
+
+1. Log in to the [Auth0 Dashboard](${manage_url}).
+2. Navigate to **Logs > Streams**.
+3. Click on a stream.
+4. Select the **Health** tab. 
+
+## Understanding Stream Health States
+
+**add some stuff here**
+
+Status types:
+
+| Status | Description |
```

Comment:
I think these should be described in a user friendly way in terms of scenarios. Maybe like what we've been talking about in slack?

https://auth0.slack.com/archives/CEJHNE5QV/p1585588481287000?thread_ts=1585584951.281100&cid=CEJHNE5QV